src/main/java/dk/bec/polonez/reservationsystem/dto/notificationDto/ReservationConfirmationInput.java

```diff
@@ -0,0 +1,11 @@
+package dk.bec.polonez.reservationsystem.dto.notificationDto;
```

Comment:
**advice** Wrong package name, upper-case in ```notificationDto``` is against Java package naming conventions.